jupyter_server_documents/rooms/yroom_file_api.py

```diff
@@ -319,9 +319,17 @@ async def _load_content(self, jupyter_ydoc: YBaseDoc) -> None:
             self.log.info(f"Processing outputs for loaded notebook: '{self.room_id}'.")
             file_data = self.outputs_manager.process_loaded_notebook(file_id=self.file_id, file_data=file_data)
 
+        # Replace CRLF line terminators with LF line terminators
+        # Fixes #176, see issue description for more context.
+        content: str = file_data['content']
```

Comment:
Is this guaranteed to be always str?

Reply:
Good catch. I think this should always be a string, but there may be some weird scenario where this is an instance of `bytes`.

I can use the `isinstance()` function to prevent any weird edge case.

Reply:
Lemme work on that now